db/seeds/08_full_stack_track.rb

```diff
@@ -7,8 +7,18 @@
   Track.delete_all
 end
 
-Track.create!(
+create_or_update_track(
   title: "Full Stack",
   description: "The OG Odin",
-  position: 1
+  position: 1,
+  courses: [
```

Comment:
@mindovermiles262 The track courses created don't have their `track_position`s set. Can you add those?